Doc/library/os.rst

```diff
@@ -968,6 +968,9 @@ as internal buffering of data.
    <fd_inheritance>` by default or non-inheritable if *inheritable*
    is ``False``.
 
+   If *fd* is equal to *fd2*, the file descriptor is validated, but its
+   inheritability is not changed unless *inheritable* is ``False``.
+
```

Comment:
Regarding [comment from PR #5713](https://github.com/python/cpython/pull/5713#discussion_r321954642), I'm not sure if it should be `versionchanged`. The current mess is unintentional, so it's not like we had some clear behavior before and then we changed it.